controllers/runner_controller.go

```diff
@@ -834,6 +834,7 @@ func newRunnerPodWithContainerMode(containerMode string, template corev1.Pod, ru
 		if dockerdContainer != nil {
 			template.Spec.Containers = append(template.Spec.Containers[:dockerdContainerIndex], template.Spec.Containers[dockerdContainerIndex+1:]...)
 		}
+		// TODO: What was the intention here?
```

Comment:
Can you help me with this one?

```
controllers/runner_controller.go:838:6: SA4000: identical expressions on the left and right side of the '<' operator (staticcheck)
                if runnerContainerIndex < runnerContainerIndex {
```

Reply:
Good catch! See https://github.com/actions-runner-controller/actions-runner-controller/pull/1794/files#r974803081